app/controllers/EndDateQuestionController.scala

```diff
@@ -44,13 +44,18 @@ class EndDateQuestionController @Inject()(actionsProvider: ActionsProvider,
   def show(taxYear: Int,
            benefitType: BenefitType,
            sessionDataId: UUID): Action[AnyContent] = actionsProvider.endOfYearSessionDataFor(taxYear, benefitType, sessionDataId) { implicit request =>
-    Ok(pageView(EndDateQuestionPage(taxYear, benefitType, request.stateBenefitsUserData, formsProvider.endDateYesNoForm(taxYear))))
+    Ok(pageView(EndDateQuestionPage(
+      taxYear,
+      benefitType,
+      request.stateBenefitsUserData,
+      formsProvider.endDateYesNoForm(taxYear, request.stateBenefitsUserData.claim.get)
+    )))
   }
 
   def submit(taxYear: Int,
              benefitType: BenefitType,
              sessionDataId: UUID): Action[AnyContent] = actionsProvider.endOfYearSessionDataFor(taxYear, benefitType, sessionDataId).async { implicit request =>
-    formsProvider.endDateYesNoForm(taxYear).bindFromRequest().fold(
+    formsProvider.endDateYesNoForm(taxYear, request.stateBenefitsUserData.claim.get).bindFromRequest().fold(
```

Comment:
You only need to pass the startDate here, not the whole object.

Reply:
hm I was taking as a reference what was in the `taxTakenOffYesNoForm`, which was behaving similarly?